src/main/java/com/openmrs/migrator/core/utilities/KettleUtils.java

```diff
@@ -0,0 +1,84 @@
+package com.openmrs.migrator.core.utilities;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class KettleUtils {
+
+  private static Logger log = LoggerFactory.getLogger(KettleUtils.class);
+
+  private final String KETTLE_PROPERTIES = "kettle.properties";
+
+  private final String KETTLE_DIR = ".kettle";
+
+  public void loadProperties() {
```

Comment:
This loadProperties method should be part of the PDI service.

I don't think we need to have a KettleUtilities class - kettle is PDI so this falls under the scope of PDI

Reply:
done:  moved the logic of  KettleUtils to PDI service